tox.ini

```diff
@@ -0,0 +1,44 @@
+[tox]
+envlist = py37
+isolated_build = True
+
+[tox:.package]
+basepython = 3
+
+[testenv:unittests]
```

Comment:
I'd change this to just `[testenv]` that way it's the default when simply running `tox` without any arguments.